kevm-pyk/src/kevm_pyk/kdist/plugin.py

```diff
@@ -96,13 +96,5 @@ def _build_env(self) -> Iterator[Path]:
             'syntax_module': 'ETHEREUM-SIMULATION',
         },
     ),
-    'foundry': KEVMTarget(
```

Comment:
Will we still be able to build `kontrol` if we remove the `foundry` target from `kevm-dist`? Is there a way to build Kontrol locally other than `poetry -C kevm-pyk run kevm-dist build foundry`?

Reply:
I understand we have to separate the building of Kontrol from KEVM especially considering that `foundry.md` will be moved to Kontrol, but I'm wondering if it's safer (or makes sense at all) to split this change into two stages: delete `kontrolx` first, and then move `foundry.md` to Kontrol while also removing `foundry` target from `kevm-dist`.

Reply:
I agree, in order to make the downstream dependency update as simple as possible, kompilation related changes, including changes to the kompilation CLI, should be taken care of separately: https://github.com/runtimeverification/kontrol/issues/61. This PR should focus on removing `kontrolx` and corresponding tests: https://github.com/runtimeverification/evm-semantics/issues/2113.